cmb2-conditionals.php

```diff
@@ -91,6 +91,9 @@ public function __construct() {
 			add_action( 'admin_init', array( $this, 'admin_init' ), self::PRIORITY );
 			add_action( 'admin_footer', array( $this, 'admin_footer' ), self::PRIORITY );
 
+			// Enable conditionals on front-end if using recent version of CMB2.
+			add_action( 'cmb2_footer_enqueue', array( $this, 'cmb2_footer' ), self::PRIORITY );
```

Comment:
I've never used CMB2 Conditionals, and I'm not sure about the context of when this hook happens, but I would make sure (maybe within the `cmb2_footer` callback) that you only enqueue the script if it is known for sure that is needed.